improver_tests/wxcode/wxcode/test_wxcode_decision_tree.py

```diff
@@ -37,6 +37,7 @@
 from improver.wxcode.utilities import expand_nested_lists
 from improver.wxcode.wxcode_decision_tree import (
     START_NODE, wxcode_decision_tree)
+from . import check_diagnostic_lists_consistency
```

Comment:
I think the `expand_nested_lists` function no longer needs to be imported. This also applies for the test_wxcode_decision_tree_global.py file.

Reply:
Agreed - removed.